demo/src/main/java/rtmp_client/RtmpClient.java

```diff
@@ -0,0 +1,24 @@
+package net.butterflytv.rtmp_client;
```

Comment:
We can't accept a change that includes a random package like this. Where does this file actually originate from? If you copied it from some other project, note that you can't submit it here claiming to be the author unless you were also the original author of the original file. Please clarify. Thanks!